docker-compose-confluent-cloud.yml

```diff
@@ -0,0 +1,22 @@
+version: '2'
+services:
+  ppmbsm:
+    build:
+      context: .
+      dockerfile: Dockerfile.cdot
+    environment:
+      DOCKER_HOST_IP: ${DOCKER_HOST_IP}
+      KAFKA_TYPE: ${KAFKA_TYPE}
+      CONFLUENT_KEY: ${CONFLUENT_KEY}
+      CONFLUENT_SECRET: ${CONFLUENT_SECRET}
+      PPM_CONFIG_FILE: ppmBsm.properties
+  ppmtim:
+    build:
+      context: .
+      dockerfile: Dockerfile.cdot
+    environment:
+      DOCKER_HOST_IP: ${DOCKER_HOST_IP}
+      KAFKA_TYPE: ${KAFKA_TYPE}
+      CONFLUENT_KEY: ${CONFLUENT_KEY}
+      CONFLUENT_SECRET: ${CONFLUENT_SECRET}
+      PPM_CONFIG_FILE: ppmTim.properties
```

Comment:
Since we specified the Dockerfile.cdot as our dockerfile values, should we also use the corresponding cdot properties files? Or maybe swap out Dockerfile.cdot to the default Dockerfile?

Reply:
The default dockerfile is now being utilized here.